guides/slack.md

```diff
@@ -0,0 +1,67 @@
+ 
+## Mandatory channels
+
+As of writing, it is expected every slack user will be present in `#announcements`, this is so that everyone can be reached in case of an important announcement.
+There are no other channels mandatory for _all_ users, but it is expected you will be in channels relevant to your working group.
+
+## Private channels and group chats
+
+Private channels can be used for specific tasks and group chats may be used to have small private discussions if necessary. It's worth noting that everything posted on our slack is logged and could be subject to a Freedom of Information request (specifically for channels relating to public sector projects), so it is wise to always use slack as if your message could be read.
+ 
+# Notifications and Threads
+ 
+It's generally expected you'll be available in relevant, non-optional channels during working hours - this means project/group channels you're directly involved in and `#announcements`. When minimising notification noise, it's entirely reasonable for you to set availability with the teams you're working with. All other optional channels are exactly that, optional. It is also reasonable to mute slack entirely and check periodically, whatever works best for you, remembering that slack is the primary method of communication within the company while remote.
+
+## Threads
+
+We don't have a set etiquette regarding threads, though generally we try to keep specific discussions in general channels to threads. Some notable exceptions currently are `announcements`, and `help`, where threading posts is actively encouraged. Any channels with specifc threading (or other) requirements will describe this in the channel topic.
+
+## @ Usage
+
+### @user
+You use @user to get a specific user's attention in any channel where you'd like to ensure they get notified about something important.
+
+### @here
+Using @here will notify every user who is in the channel and active, for company wide mandatory channels this will usually be used to ensure important announcements are flagged. In other channels, feel free to use @here if you believe the post is immediately relevant to every user present. Some channels can get '@here noisy' such as `#help` where people are seeking to generate awareness, it is entirely reasonable to leave or mute such optional channels.
+
+### @channel
+Using @channel will notify everyone in the channel who is present, and bypasses some settings such as muting the channel. This is only used for messages of the utmost importance and should generaly be used sparingly.
+
+# Conduct
+
+We have never outlined a specific set of rules of conduct and behaviour on slack as we have never needed to, please remember to be courteous, polite and treat your colleagues with respect, and while off topic and non-work chat is encouraged, please remember to keep such chat work friendly!
```

Comment:
I'd argue that this document is the first draft of our code of conduct, and that we're probably writing it a little after it was needed! I don't think I would say "we didn't need a code before".

Reply:
I hadn't thought of it that way - will remove